playbooks/roles/rabbitmq/tasks/main.yml

```diff
@@ -163,21 +163,32 @@
     user: guest
     state: absent
 
-- name: add rabbitmq vhost
+# TODO: what is this used for?
+- name: ensure vhost /test is present
   rabbitmq_vhost:
-    name: "{{ RABBITMQ_VHOST }}"
-    node: "rabbit@{{ RABBITMQ_HOSTNAME }}"
+    name: /test
```

Comment:
Could just be a basic test of rabbitmq's functionality.. I think it's ok to leave in.